src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2360,11 +2380,11 @@ private void PopulateInterfaceImplTableRows()
             foreach (ITypeDefinition typeDef in this.GetTypeDefs())
             {
                 var typeDefHandle = GetTypeDefinitionHandle(typeDef);
-                foreach (ITypeReference interfaceRef in typeDef.Interfaces(Context))
```

Comment:
@tmat We're considering this for Ask mode so I'd like to keep this change as focused as possible for now.